cfy_manager/components/cli/cli.py

```diff
@@ -138,9 +136,3 @@ def _deploy_external_cert(self):
             return EXTERNAL_CA_CERT_PATH
         else:
             return EXTERNAL_CERT_PATH
-
-    def replace_certificates(self):
```

Comment:
I'm not sure I understand why this isn't required any more?

Reply:
Well, I found out in the tests I did that this is not needed because the CLI would still use the same certificates' locations, so there's no need to configure them again. It's not related to this PR specifically, but I thought I might as well change it now. Maybe I should make a different PR for it?

Reply:
Fair enough, makes sense!